node/overseer/src/memory_stats.rs

```diff
@@ -14,7 +14,10 @@
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
-use tikv_jemalloc_ctl::{epoch, stats, Error};
+// Allow unused code as they might be enabled only for the jemalloc-stats feature flag.
+#![allow(dead_code)]
```

Comment:
Instead of allowing dead code, you can gate the module on conditional compilation as you do elsewhere, i.e. with

```
#[cfg(any(target_os = "linux", feature = "jemalloc-stats"))]
```

Reply:
I didn't expect this to work because i tried something similar before. But it worked nicely. So i've pushed the update.

Reply:
Remember to remove the commented-out allow.

Reply:
Yeah, i've removed it. Thanks for pointing it out